moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -226,8 +211,31 @@ bool Servo::updateParams()
       }
       if (params.move_group_name != servo_params_.move_group_name)
       {
-        RCLCPP_INFO_STREAM(LOGGER, "Move group changed from " << servo_params_.move_group_name << " to "
-                                                              << params.move_group_name);
+        if (servo_status_ == StatusCode::PAUSED)
+        {
+          invalid_parameter_update_ = false;
```

Comment:
`invalid_parameter_update_` is synonymous with `servo_status != StatusCode::PAUSED` now, so no need for this member variable, right?